client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -155,12 +175,39 @@ const CheckoutThankYou = React.createClass( {
 		} );
 
 		let purchases = null,
-			wasJetpackPlanPurchased = false;
+			wasJetpackPlanPurchased = false,
+			wasOnlyDotcomPlanPurchased = false;
 		if ( this.isDataLoaded() && ! this.isGenericReceipt() ) {
 			purchases = getPurchases( this.props );
 			wasJetpackPlanPurchased = purchases.some( isJetpackPlan );
+			wasOnlyDotcomPlanPurchased = purchases.every( isPlan );
+		}
+
+		const userCreatedMoment = moment( this.props.user.date );
+		const isNewUser = userCreatedMoment.isAfter( moment().subtract( 2, 'hours' ) );
+		const isInAbTest = abtest( 'paidNuxStreamlined' ) === 'streamlined';
+
+		// placeholder
+		if ( ! this.isDataLoaded() && ! this.isGenericReceipt() ) {
```

Comment:
I think we should extract the placeholder logic/condition to a standalone method and use that everywhere because it is different here than on this line: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/checkout-thank-you/index.jsx#L154